todo_app/templates/item.html

```diff
@@ -0,0 +1,26 @@
+{% extends "layout.html" %}
+{% block title %}To Do Item{% endblock %}
+
+{% block content %}
+<div class="jumbotron">
+    <h1 class="display-4">To-Do App</h1>
+    <p class="lead">Just another to-do app.</p>
+  </div>
+
+  <form action="/submit_item/{{ToDo.id}}" method="post">
+      <div>
+          <input type="text" name="title" id="title" value="{{ToDo.title}}" required>
+      </div>
+      <div>
+          <input type="radio" name="status" id="status" value="Not Started" {% if not_started %} checked = "checked" {% endif %}/>Not Started
```

Comment:
You have the same `id` for all three radio button elements. An HTML element's id should be unique on the whole page.

`name` is what controls its key in the form so you need to keep that the same for all of them, but you should either remove the id (nothing references it) or make them unique.